yolo/json/yolov6.json

```diff
@@ -0,0 +1,108 @@
+{
+    "nn_config":
+    {
+        "output_format" : "detection",
+        "NN_family" : "YOLO",
+        "input_size": "416x416",
+        "NN_specific_metadata" :
+        {
+            "classes" : 80,
+            "coordinates" : 4,
+            "anchors" : [10,13, 16,30, 33,23, 30,61, 62,45, 59,119, 116,90, 156,198, 373,326],
+            "anchor_masks" :
+            {
+                "side52" : [0,1,2],
```

Comment:
These aren't needed anymore, might be confusing, but needed for the demo app.
Therefore let's still define them but leave them empty:
```
            "anchors" : [],
            "anchor_masks" :
            {
            },
```